lib/src/render/find_replace_menu/find_menu_service.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:appflowy_editor/src/render/find_replace_menu/find_replace_widget.dart';
+import 'package:flutter/material.dart';
+import '../../editor_state.dart';
+
+abstract class FindReplaceService {
+  void show();
+  void dismiss();
+}
+
+class FindReplaceMenu implements FindReplaceService {
+  FindReplaceMenu({
+    required this.context,
+    required this.editorState,
+    required this.replaceFlag,
+  });
+
+  final BuildContext context;
+  final EditorState editorState;
+  final bool replaceFlag;
+  final double topOffset = 52;
+  final double rightOffset = 40;
+
+  OverlayEntry? _findReplaceMenuEntry;
+  bool _selectionUpdateByInner = false;
+
+  @override
+  void dismiss() {
+    if (_findReplaceMenuEntry != null) {
```

Comment:
No need to go through all of this logic if for some reason `dismiss()` is called without there being a current entry.
```suggestion
    if (_findReplaceMenuEntry == null) {
      return;
    }

    if (_findReplaceMenuEntry != null) {
```